pkg/webhook/validating/webhook.go

```diff
@@ -70,11 +70,13 @@ func NewWebhook(cfg WebhookConfig, validator Validator, ot opentracing.Tracer, r
 	// Create our webhook and wrap for instrumentation (metrics and tracing).
 	return &instrumenting.Webhook{
 		Webhook: &staticWebhook{
-			objType:      helpers.GetK8sObjType(cfg.Obj),
-			deserializer: codecs.UniversalDeserializer(),
-			validator:    validator,
-			cfg:          cfg,
-			logger:       logger,
+			objType:         helpers.GetK8sObjType(cfg.Obj),
+			deserializer:    codecs.UniversalDeserializer(),
+			validator:       validator,
+			cfg:             cfg,
+			logger:          logger,
+			metricsRecorder: recorder,
```

Comment:
I had to add the recorder here aswell to be able to use it inside the `Review` method of the validating webhook implementation. I'm not 100% sure of this so please review this bit carefully.